packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2537,6 +2559,12 @@ export class ContainerRuntime
 				"entryPoint must be defined on data store runtime for using getAliasedDataStoreEntryPoint",
 			);
 		}
+		this.garbageCollector.nodeUpdated(
```

Comment:
Shouldn't this code path throw as well?

Reply:
No, it should not because aliased data stores can never be tombstoned. We should however make this impplicit in telemetry.

Reply:
If there were a bug that resulted in an aliased datastore being marked as Unreferenced, I think we'd want Tombstone to reflect Sweep - it would be deleted.  Hard to imagine, I'll admit.

Reply:
I agree. With the new code, this will throw an error. We should indicate in the error or telemetry that this is a aliased data store and this should never have been tombstoned.